### PR TITLE
New package: 9mount-1.3

### PR DESCRIPTION
**Pull request by CMB**

### CODE REVIEW

**Comment by ebfe** [comment]
Consider adding the follwing upstream patch http://sqweek.net/hg/9mount/index.py/rev/c6da9cac8d01. Otherwise working with hosts that have a AAAA record is a bit annoying (v9fs only supports ipv4).

**Comment by CMB** [comment]
Ok, I added that patch.
It's really too bad that V9FS doesn't do IPv6.  Do you know if there are any
plans to change that in the near future?

**Comment by ebfe** [comment]
> It's really too bad that V9FS doesn't do IPv6.  Do you know if there are any
> plans to change that in the near future?

Don't think so. In some way it actually supports ipv6 and a whole lot of
other things via the fd transport (rfdno/wdfno) but that isn't supported
by 9mount.

File: srcpkgs/9mount/INSTALL [hunk 1]
```diff
@@ -0,0 +1,5 @@
+case "${ACTION}" in
+post)
+	chown root:users usr/bin/9{mount,umount,bind}
```
Comment (ebfe) [inline srcpkgs/9mount/INSTALL]:
```
9mount-1.3_1: configuring ...
chown: cannot access 'usr/bin/9{mount,umount,bind}': No such file or directory
chmod: cannot access 'usr/bin/9{mount,bind,umount}': No such file or directory
```

Comment (ebfe) [inline srcpkgs/9mount/INSTALL]: That's due to the 'sh' alternative being set to 'dash' (which seems to be the default). For reasons unknown it's set to 'bash' in the masterdir, so to miss things like that.

**Comment by CMB** [comment]
Yeah apparently usr/bin/9{bind,mount,umount} was a bashism and your
/bin/sh is dash (or busybox sh for that matter).
Fixed.  Thanks for catching.

**Comment by ebfe** [comment]
Thanks.

**Pull request merged**